crates/ln-dlc-node/src/ln/event_handler.rs

```diff
@@ -251,7 +46,6 @@ where
 /// don't require custom behaviour
 pub mod handlers {
     use super::*;
```

Comment:
🔧 I think it's not recommended to use this kind of import outside of test modules.

Reply:
why would this not be recommended? it's perfectly valid syntax, and used for, well... organising the code? Happy to learn more about the reasons if you don't mind sharing some links.

Moved it into `common_handlers.rs`.

Reply:
@klochowicz: There's a Clippy lint that explains this: https://rust-lang.github.io/rust-clippy/master/index.html#/wildcard_imports.

(Unresolved conversation for visibility.)

Reply:
thank you! 

....shall we enable it then? :)

Reply:
aaaaaa I think we were talking past each other - I thought your comment was about having a module defined inside module (with `mod handlers {}`, and you were talking about wildcard imports!

Reply:
Oh lol. I actually did not mind the internal submodule.